packages/web3-utils/src/converters.ts

```diff
@@ -383,3 +400,109 @@ export const toChecksumAddress = (address: Address): string => {
 	}
 	return checksumAddress;
 };
+
+/**
+ *  used to flatten json abi inputs/outputs into an array of type-representing-strings
+ */
+export const flattenTypes = (includeTuple: boolean, puts: Components[]): string[] => {
```

Comment:
Does this code relates to this PR?

Reply:
I didn't add this, it was merged in a [separate PR](https://github.com/ChainSafe/web3.js/pull/4535/files#diff-217301804b75cf6f71d5a808105e97fd258371e0846a38f44841668b5c852148R399), but I'm not sure why it's showing it as an addition here